homeassistant/components/freebox/__init__.py

```diff
@@ -59,7 +75,9 @@ async def async_reboot(call: ServiceCall) -> None:
 
     async def async_close_connection(event):
         """Close Freebox connection on HA Stop."""
-        await router.close()
+        await api.close()
+        unsub_dispatcher()
+        hass.services.async_remove(DOMAIN, SERVICE_REBOOT)
```

Comment:
@MartinHjelmare I refactored the API initialisation routine, but I am a little unfamiliar with the unload routine.
Is this the correct way to do it?